src/catalogue/category/catalogueCard.component.tsx

```diff
@@ -1,7 +1,7 @@
 import DeleteIcon from '@mui/icons-material/Delete';
 import EditIcon from '@mui/icons-material/Edit';
 import MoreHorizIcon from '@mui/icons-material/MoreHoriz';
-import SaveAsIcon from '@mui/icons-material/SaveAs';
+import DuplicateIcon from '@mui/icons-material/SaveAs';
```

Comment:
Technically this is fine as its a default export, but I would prefer it matches what MUI icons gives you when you look them up, we use their naming everywhere even if it isn't used as a save as icon here.

Reply:
Noted, just pushed a commit undoing those changes.